geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -38,6 +39,114 @@ app.module.directive('appMymaps', app.mymapsDirective);
  * @constructor
  * @export
  */
-app.MymapsDirectiveController = function() {};
+app.MymapsDirectiveController = function() {
+
+  /**
+   * @type {string}
+   * @private
+   */
+  this.defaultTitle_ = 'Map without title';
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.modifying = false;
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.choosing = false;
+
+  /**
+   * @type {Object?}
```

Comment:
But to avoid renaming-related problems and have more explicit variables declared in the constructor I'd have used `this.mapTitle` and `this.mapId` properties.